generators/dart/variables.js

```diff
@@ -16,16 +16,16 @@ const {NameType} = goog.require('Blockly.Names');
 
```

Comment:
nit: we usually use 2 blank lines after the import statements... feels like something clang-format should automate though